CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 22.08
+
+* The `config` endpoint is now accessible in read-only mode by:
+
+  * `master_tenant`
+
+* The `status` endpoint has now a new route:
+
+  * `/config`
```

Comment:
What does it mean?

Reply:
It means that `status` plugin takes as a parameter the `config` dict; from which we check if the `master_tenant_uuid` key is set, the status of `master_tenant` is set to `Ok`; otherwise to `fail`: [wazo-webhookd-status](https://github.com/wazo-platform/wazo-webhookd/blob/master/wazo_webhookd/plugins/status/http.py#L12)